documentation/faq/README.md

```diff
@@ -0,0 +1,24 @@
+# Frequently asked questions
+
+- [What Operating Systems does the WoTT agent support?](#1)
```

Comment:
@fshmcallister these anchor links are important for SEO, so please name them something more descriptive than numbers.

Reply:
changed to be more descriptive

Reply:
Let me know if you can think of any other pressing questions